packages/gluestick-cli/new.js

```diff
@@ -90,5 +91,15 @@ module.exports = (appName, options, exitWithError) => {
           stdio: 'inherit',
         },
       );
+    })
+    .catch((error) => {
+      rimraf(
+        // Make sure CWD includes appName, we don't want to remove other files
+        process.cwd().includes(appName) ? process.cwd() : path.join(process.cwd(), appName),
```

Comment:
this is just check if cwd contains appName, because of this line above:
```
process.chdir(appName);
```
so this just ensures it won't delete things we don't want